schema-based-on-diagram.sql

```diff
@@ -0,0 +1,40 @@
+CREATE TABLE medical_histories (
+  id SERIAL PRIMARY KEY,
+  admitted_at TIMESTAMP NOT NULL,
+  patient_id INTEGER REFERENCES patients(id),
+  status VARCHAR(255) NOT NULL
+);
+
+CREATE TABLE patients (
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(255) NOT NULL,
+  date_of_birth DATE NOT NULL
+);
+
+CREATE TABLE treatments (
+  id SERIAL PRIMARY KEY,
+  type VARCHAR(255) NOT NULL,
+  name VARCHAR(255) NOT NULL
+);
+
+CREATE TABLE invoices (
+  id SERIAL PRIMARY KEY,
+  total_amount DECIMAL(10, 2) NOT NULL,
+  generated_at TIMESTAMP NOT NULL,
+  medical_history_id INTEGER REFERENCES medical_histories(id)
+);
```

Comment:
- [ ] You missed one entity(column) for this table `payed_at`, Please make sure to add that to complete the table implementation.

![image](https://user-images.githubusercontent.com/83803414/230098945-410051e6-3402-46ee-b1c7-902bd52c0f5e.png)